filament/backend/include/private/backend/Program.h

```diff
@@ -46,7 +47,12 @@ class Program {
         bool strict = false;        // if true, this sampler must always have a bound texture
     };
 
-    using SamplerGroupInfo = std::array<utils::FixedCapacityVector<Sampler>, BINDING_COUNT>;
+    struct SamplerGroup {
```

Comment:
There is another class called `SamplerGroup` so it would be better to use another name, maybe `SamplerGroupData`.

Reply:
It's done.